src/styles/blocks/error.scss

```diff
@@ -0,0 +1,48 @@
+.error {
+  position: relative;
+
+  box-sizing: inherit;
+  padding: 1.25rem 2.5rem 1.25rem 1.5rem;
+  border-radius: 0.38em;
+
+  background-color: #f14668;
+  color: #fff;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+
+  &__delete {
+    width: 20px;
+    height: 20px;
+    border: none;
+    border-radius: 50%;
+
+    background-color: rgba(10, 10, 10, 0.2);
+    cursor: pointer;
+
+    &::before {
+      width: 50%;
+      height: 2px;
+    }
+
+    &::after {
+      width: 2px;
+      height: 50%;
+    }
+
+    &::before,
+    &::after {
+      position: absolute;
+
+      content: "";
+      background-color: #fff;
+      transform: translateX(-50%) translateY(-50%) rotate(45deg);
+    }
+  }
+
+  & > &__delete {
```

Comment:
Avoid using the '&' selector in this way. It's better to use the parent class name instead.

Reply:
Changed it in this way. Is it correct?